package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "codefresh",
-  "version": "0.88.6",
+  "version": "0.88.7",
```

It will be the new feature, so should be minor version bump.
```suggestion
  "version": "0.89.0",
```